### PR TITLE
add sync task

### PR DESCRIPTION
**Pull request by bjfletcher**
It worked. Phew! I thought I'd make it easier by turning it into a `make sync`. Feel free to drop this if you don't like the look of it. :)

### CODE REVIEW

**Comment by bjfletcher** [comment]
image

File: health/db-sync-state.js [hunk 1]
```diff
@@ -37,20 +37,7 @@ module.exports = exports = new (class DBSyncStateCheck extends nHealthCheck {
...
-			--password \${PRODUCTION_DATABASE_PASSWORD}
-			--host \${PRODUCTION_DATABASE_HOST}
-			--port \${PRODUCTION_DATABASE_PORT}
-			--dbname \${PRODUCTION_DATABASE_NAME}
-			--command 'select syndication.reload_all();'
-\`\`\`
-
-Substituting the above \`\${PRODUCTION_DATABASE_*}\` placeholders with the correct values from vault.`,
-/*eslint-enable*/
+	panicGuide: 'you can sync them with the repo as follows: 1. ngage get-config --env prod 2. make sync',
```
Comment (leggsimon) [inline health/db-sync-state.js]: This should explain that you should clone and cd into the repo first I think

Comment (bjfletcher) [reply health/db-sync-state.js]: @leggsimon good point, let's not merge this PR... I think maybe it'll be nice to have a check early in `n-gage` to check whether .env is production, and prompt the user "is this what you want to load?"

**Comment by leggsimon** [comment]
Thanks for this @bjfletcher but I'm going to close this in favour of #75 and would prefer to keep the `psql` command over `make` as it doesn’t require someone to clone the repo to run. Feel free to re-open if you have concerns.

**Pull request closed without merging**